src/drt/src/db/obj/frInst.h

```diff
@@ -94,6 +95,15 @@ class frInst : public frRef
   void setOrigin(const Point& tmpPoint) override { xform_.setOffset(tmpPoint); }
   dbTransform getTransform() const override { return xform_; }
```

Comment:
Is this used anywhere now?

Reply:
Unfortunatly, yes. `getTransform()` itself is used in the now renamed `getNoRotationTransform()` and in some other cases. Both `setOrigin()` and `setOrient()` still se some uses. I intend to completely eliminate these two functions, but I'm not sure it is possible. (I'm not 100% sure about what I'm about to say) During Pin Access virtual vias have to be created to check for DRVs when considering if a Access Point is valid. These vias exists only on the drt, and don't have an odb counterpart during that checking so they currently need to use and configure the drt Transform.

Reply:
Does it need to be public?